sky/packages/sky/lib/src/material/material_app.dart

```diff
@@ -73,18 +75,24 @@ class _MaterialAppState extends State<MaterialApp> {
   }
 
   Widget build(BuildContext context) {
-    return new Theme(
-      data: config.theme ?? new ThemeData.fallback(),
-      child: new DefaultTextStyle(
-        style: _errorTextStyle,
-        child: new DefaultAssetBundle(
-          bundle: _defaultBundle,
-          child: new Title(
-            title: config.title,
-            child: new Navigator(
-              key: _navigator,
-              routes: config.routes,
-              onGenerateRoute: config.onGenerateRoute
+    return new SizeObserver(
+      onSizeChanged: (Size size) => setState(() { _size = size; }),
```

Comment:
I think we actually need the size of the app, people are going to want to set breakpoints.